doc/classes/NavigationPolygon.xml

```diff
@@ -172,6 +172,7 @@
 	<members>
 		<member name="agent_radius" type="float" setter="set_agent_radius" getter="get_agent_radius" default="10.0">
 			The distance to erode/shrink the walkable surface when baking the navigation mesh.
+			[b]Note:[/b] The radius must be positive. If the radius is [code]0.0[/code], it won't be possible to fix invalid outline overlaps and other precision errors during the baking process. As a result, some obstacles may be excluded incorrectly from the final navigation mesh, or may delete the navigation mesh's polygons.
```

Comment:
Is 0 allowed or not? The description is unclear ("positive" in mathematics means "greater than zero", if zero is permitted the term is "nonnegative"), the mention of zero here is misleading if it's not permitted, better to be explicit if `>= 0` is meant

Reply:
0 is allowed. You're right, 0 is neither positive nor negative. Thanks for catching that.